### ISSUE TITLE
Image carousel pause autorotation is broken

### ISSUE THREAD

**Issue by Grecco-O** [manyfold3d/manyfold#3639]
The pause button on the image autorotation doesn't works.

**To Reproduce**
1. Go into a model
2. Click on the pause button
3. See error / See images not pausing
